src/EventStore.Projections.Core/Services/Management/ProjectionManager.cs

```diff
@@ -1304,5 +1307,40 @@ public PendingProjection(long projectionId, ProjectionManagementMessage.Command.
 					replyEnvelope, DefaultProjectionExecutionTimeout);
 			}
 		}
+
+		public void Handle(TelemetryMessage.Request message) {
+			var customProjectionCount = 0;
+			var customProjectionRunningCount = 0;
+			var standardProjectionCount = 0;
+			var standardProjectionRunningCount = 0;
+
+			foreach (var proj in _projections.Values) {
+				if (proj.Mode != ProjectionMode.Continuous)
```

Comment:
Are we not interested in whether users are running queries or one-time projections?

Reply:
My rationals were they would be quite hard to capture live anyway because we only query the manager once day. Also, they cause to ramp up artificially the `totalCount` telemetry too.